lib/utils/strip-quotes.ts

```diff
@@ -1,10 +1,19 @@
-export function stripQuotes(value?: string | null) {
-  const trimmed = value?.trim();
+/**
+ * Strip leading/trailing quotes from a string and handle null values.
+ */
+export function stripQuotes(value?: string | null): string | null {
+  if (value === null || value === undefined) {
+    return null;
+  }
```

Comment:
Can probably be removed, since all vendors' `parseDefaultValue` already include that check ?

Reply:
It's used by cockroach outside of a non-`parseDefaultValue` function. 👍